### PR TITLE
feat(send-signal): added documentation about command send-signal

### LINKED ISSUE #1877: Update cli documentation

### PR DESCRIPTION
**Pull request by QuentinEscudierScalingo**
Must be merged after: https://github.com/Scalingo/cli/pull/841

Fix #1877

### CODE REVIEW

**Review by curzolapierre — APPROVED** [review]
LGTM

File: src/changelog/api/_posts/2023-01-05-send-signal.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+modified_at: 2021-09-08 10:00:00
```
Comment (EtienneM) [inline src/changelog/api/_posts/2023-01-05-send-signal.md:2]: Wrong date

File: src/changelog/api/_posts/2023-01-05-send-signal.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+title: "Add an endpoint to get cron tasks of an app"
```
Comment (EtienneM) [inline src/changelog/api/_posts/2023-01-05-send-signal.md:3]: Wrong title

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: src/changelog/cli/_posts/2023-01-05-1-27-0.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+[https://cli.scalingo.com](https://cli.scalingo.com)
+
+### Changelog
+
+
+* fix(commands): display help usage message if needed #835
+* feat(logs): allow DB type aliases #829
+* feat(db-console): add influx-console and mongodb-console aliases #830
+* feat(send-signal): send-signal command added #833
+* feat(send-signal): Allow to send signals to multiple containers via its type #841
```
Comment (EtienneM) [inline src/changelog/cli/_posts/2023-01-05-1-27-0.md:18]:
```suggestion
### Changelog

* fix(commands): display help usage message if needed
* feat(logs): allow DB type aliases
* feat(db-console): add influx-console and mongodb-console aliases
* feat(send-signal): send-signal command added
* feat(send-signal): Allow to send signals to multiple containers via its type
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: src/_posts/platform/cli/2000-01-01-features.md [hunk 4]
```diff
@@ -184,16 +184,18 @@ scalingo -a my-app pgsql-console
 scalingo -a my-app db-tunnel MONGO_URL
 ```
 
-## Manage containers, scale
+## Manage containers, scale, send signals
```
[the remark this answers is not in the document]
Comment (QuentinEscudierScalingo) [reply src/_posts/platform/cli/2000-01-01-features.md]: I agree for the `s` but not on the new line, this is not homogeneous with the rest of the file

[the remark this answers is not in the document]
Comment (EtienneM) [reply src/_posts/platform/cli/2000-01-01-features.md]: But this is consistent with the rest of the doc :p 

I would update the rest of the file accordingly

[the remark this answers is not in the document]
Comment (QuentinEscudierScalingo) [reply src/_posts/platform/cli/2000-01-01-features.md]: Ok, I'm on it thx

**Review by EtienneM — APPROVED** [review]

**Review by curzolapierre — APPROVED** [review]

**Pull request merged**